### DISCUSSION TITLE
cropperJsInterop.min.js not found

### DISCUSSION THREAD

**Discussion by emrebiber** [CropperBlazor/Cropper.Blazor#292] [Q&A] [answered]
Hello friends,

When I deploy my Blazor server app to IIS I'm getting this error below.
![image](https://github.com/CropperBlazor/Cropper.Blazor/assets/10804043/422492c6-ac00-4586-81fa-e7fbf30d4224)

It works fine in localhost. I followed all the steps in the installation guide. 

- added Cropper.Blazor Nuget package
- added @using Cropper.Blazor.Components statement in _Imports.razor
- added Cropper.Blazor/cropper.min.css in _Host/cshtml
- added <script src="_content/Cropper.Blazor/cropper.min.js"></script> in _Host/cshtml
- added builder.Services.AddCropper(); in Program.cs

Can you guys please help me? Maybe I'm missing something. 

Thanks for this such an awesome tool. I appreciate all your efforts.

**Comment by MaxymGorn** [comment, marked as answer]
Hi @emrebiber. Previously we see similar problem in ISS server and you can try to use following code for overriding path to js module: #https://github.com/CropperBlazor/Cropper.Blazor/issues/142#issuecomment-1603316006 
or read Installation guide in the home page of project site about overriding global or internal path to CropperJSInterop.min.js module and so on
